app/models/Review.scala

```diff
@@ -304,23 +304,13 @@ object Review {
     proposalsCommentsCount
   }
 
-  def allVotesFromUser(reviewerUUID: String): Set[(String, Option[Double])] = Redis.pool.withClient {
+  def allVotesFromUser(reviewerUUID: String): Set[(String, Double)] = Redis.pool.withClient {
     implicit client =>
       client.smembers(s"Proposals:Reviewed:ByAuthor:$reviewerUUID").flatMap {
         proposalId: String =>
           val score = Option(client.zscore(s"Proposals:Votes:$proposalId", reviewerUUID))
           score match {
-            case None =>
```

Comment:
@nicmarti I'm not 100% sure of my change here, as I moved the result of this method from a `Set[(String, Option[Double])]` to a `Set[(String, Double)]` : tldr; votes cannot be "empty" anymore.

By reading the code, I understood that we were in the "score is empty" case only when `client.zscore()` jedis call was returning an empty score.
I'm not sure if this is related to some legacy behaviour, or if, when you vote with an "abstention", you're typically in that case (in this latest case, I think it means that I introduced a regression here)

Reply:
Reworked the change with 0461e109a0d7839144c266b7ab4f68ad76bc47b6 in order to keep the "empty score => 0 score" transformation

Reply:
If I remember well, you cannot use flatMap here, as you want to count the number of elements. I suggest we keep the existing code and we don't change this .

Reply:
@nicmarti not sure to understand which `flatMap` you talk about :-)

But anyway, doest 0461e10 look better to you ? 
I think I better captured the spirit of the initial code here, I mean transforming :
```
  def allVotesFromUser(reviewerUUID: String): Set[(String, Option[Double])] = Redis.pool.withClient {
    implicit client =>
      client.smembers(s"Proposals:Reviewed:ByAuthor:$reviewerUUID").flatMap {
        proposalId: String =>
          val score = Option(client.zscore(s"Proposals:Votes:$proposalId", reviewerUUID))
          score match {
            case None =>
              val state = Proposal.findProposalState(proposalId)
              state.flatMap {
                case ProposalState.DRAFT => None
                case ProposalState.DECLINED => None
                case ProposalState.DELETED => None
                case ProposalState.REJECTED => None
                case ProposalState.ARCHIVED => None
                case ProposalState.UNKNOWN => None
                case _ => Option((proposalId, None))
              }
            case Some(_) =>
              val state = Proposal.findProposalState(proposalId)
              state.flatMap {
                case ProposalState.DRAFT => None
                case ProposalState.DECLINED => None
                case ProposalState.DELETED => None
                case ProposalState.REJECTED => None
                case ProposalState.ARCHIVED => None
                case ProposalState.UNKNOWN => None
                case _ =>
                  Option((proposalId, score.map(_.toDouble)))
              }
          }
      }
  }
```
plus the 
```
          val sortedListOfProposals = allMyVotesForSpecificProposalType.toList.sortBy {
            case (proposalID, maybeScore) =>
              maybeScore.getOrElse(0.toDouble)
          }.reverse
```
transformation in `CFPAdmin` ....

... into :
```
  def allVotesFromUser(reviewerUUID: String): Set[(String, Double)] = Redis.pool.withClient {
    implicit client =>
      client.smembers(s"Proposals:Reviewed:ByAuthor:$reviewerUUID").flatMap {
        proposalId: String =>
          val score = Option(client.zscore(s"Proposals:Votes:$proposalId", reviewerUUID))
          score match {
            case None => None
            case Some(_) =>
              val state = Proposal.findProposalState(proposalId)
              state.flatMap {
                case ProposalState.DRAFT => None
                case ProposalState.DECLINED => None
                case ProposalState.DELETED => None
                case ProposalState.REJECTED => None
                case ProposalState.ARCHIVED => None
                case ProposalState.UNKNOWN => None
                case _ => Option((proposalId, score.map(_.toDouble).get))
              }
          }
      }
  }
```
_(+ removing transformation in `CFP` controller)_